readme.md

```diff
@@ -162,6 +162,7 @@ Thanks for contributing! 🦋🙌
 - [](# "clean-repo-tabs") [Moves the "Security" and "Insights"  to the repository navigation dropdown. Also moves the "Projects", "Actions" and "Wiki" tabs if they're empty/unused.](https://user-images.githubusercontent.com/16872793/124681343-4a6c3c00-de96-11eb-9055-a8fc551e6eb8.png)
 - [](# "repo-avatars") [Adds the profile picture to the header of public repositories.](https://user-images.githubusercontent.com/44045911/177211845-c9b0fa37-c157-4449-890e-af2602c312e3.png)
 - [](# "quick-new-issue") [Adds a link to create issues from anywhere in a repository.](https://user-images.githubusercontent.com/1402241/218251057-b94b62dd-a944-4763-b78a-fc233f7c9fd3.png)
+- [](# "small-user-avatars") [Shows a small avatar next to the username in conversation lists.](https://user-images.githubusercontent.com/44045911/230960291-721f42cc-e1ac-4fdc-83ea-2430b062f9ce.png)
```

Comment:
Can you update this in your lint PR? All screenshots should be in light mode and with an arrow ideally. Also it's best to disable nearby features to avoid confusion

<img width="489" alt="Screenshot 6" src="https://user-images.githubusercontent.com/1402241/232245640-7a3b7931-853f-4808-88f1-93b2aaf998f5.png">